### PR TITLE
Add fallback for template subscription render

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Add fallback to the configured content.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by bramkragten** [comment]
This should be fixed in the backend? We do want to evaluate a template at least once even if it doesn't have entities.

File: src/panels/lovelace/cards/hui-markdown-card.ts [hunk 1]
```diff
@@ -78,14 +78,16 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
...
+
     return html`
       <ha-card .header="${this._config.title}">
         <ha-markdown
           breaks
           class=${classMap({
             "no-header": !this._config.title,
           })}
-          .content="${this._content}"
+          .content="${content}"
```
Comment (zsarnett) [inline src/panels/lovelace/cards/hui-markdown-card.ts:90]:
```suggestion
          .content=${content}
```

**Comment by ludeeus** [comment]
> This should be fixed in the backend? We do want to evaluate a template at least once even if it doesn't have entities.

Do we? There have been changes there that now make it only evaluate if there are entities to track.

**Comment by bramkragten** [comment]
It should only apply that logic for reevaluate, what I want to just show the user name in the markdown?

**Comment by bramkragten** [comment]
Is there an issue/pr for this on the backend?

**Comment by ludeeus** [comment]
Not that I'm aware of, I started to look into it but got sidetracked

**Review by bramkragten — APPROVED** [review]

**Pull request closed without merging**